regression_tests/regression_tests.py

```diff
@@ -14,6 +14,9 @@
 import multiprocessing as mp
 from multiprocessing import Pool
 
+from tempest_commands import generate_cs_mesh, generate_ico_mesh, generate_rll_mesh
+from tempest_commands import generate_overlap_mesh, generate_offline_map, generate_test_data, apply_offline_map
```

Comment:
It may be fine to say `from tempest_commands import *`

Reply:
@rajeeja another simplification above

Reply:
Already fixed. https://github.com/ClimateGlobalChange/tempestremap/pull/91/files#diff-f44fd07fb1d6305523892c867ee23ce01ca5f5a73cf80059aab2322162719c2fR17